src/fermilib/transforms/_bksf.py

```diff
@@ -0,0 +1,430 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""bravyi_kitaev_fast transform on fermionic operators."""
+from __future__ import absolute_import
+from fermilib.ops import InteractionOperator
+import networkx
+import numpy
+from projectq.ops import QubitOperator
+
+
+def bravyi_kitaev_fast(operator):
+    """ Find the Pauli-representation of InteractionOperator for Bravyi-Kitaev
```

Comment:
While unfortunately not done consistently in FermiLib (but in ProjectQ), it is a good idea to start each doc string with a one line "title/overview" followed by a space and then a large text block which explains it in detail:

```
Super fast Bravyi Kitaev algorithm.

Find the Pauli-representation of InteractionOperator for Bravyi-Kitaev
Super fast (BKSF) algorithm. Pauli-representation of general
FermionOperator is not possible in BKSF. Also, the InteractionOperator
given as input must be Hermitian. In future we might provide a
transformation for a restricted set of fermion operator.

Args:

```